.github/workflows/deploy.yml

```diff
@@ -67,3 +67,39 @@ jobs:
           guid: '{799c0914-748b-41df-a25c-22d008f9e83f}'
           xpi_path: web-scrobbler-firefox.zip
           src_path: web-scrobbler-src.zip
+
+  deploy-edge:
+    name: Deploy Edge extension
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+
+      - name: Get Node.js version from .nvmrc
+        id: get-node-version
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '${{ steps.get-node-version.outputs.NODE_VERSION }}'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Create zipball
+        run: npx run dist chrome
```

Comment:
```suggestion
        run: npm run dist chrome
```

Reply:
Good catch! I guess we should change the Chrome one too!

Reply:
Ah yeah, missed that one